saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md

```diff
@@ -1,6 +1,6 @@
 # Identity federation & SSO
 
-- Federation lets users outside of AWS to assume temporary role (using STS) for accessing AWS resources without having a user.
+- Federation lets users outside of AWS to assume temporary role (using STS) for accessing AWS resources without having to create a user in aws.
```

Comment:
We should type AWS with capital letters ⬆️

Reply:
Agree with you.